composer.json

```diff
@@ -36,6 +36,7 @@
         "doctrine/phpcr-odm": "^1.3|^2.0",
         "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "ocramius/proxy-manager": "^1.0|^2.0",
+        "php": "^7.4||^8.0",
```

Comment:
We should not require different version of PHP for `require-dev` section.

Reply:
I'm agree with that, but I don't what is the best option, because I've added some typed properties in the unit tests (implemented in PHP 7.4) :
* update the global requirements to PHP 7.4, this will force all users to upgrade their PHP version : definitely no!
* replace the typed properties in the test Fixtures by a mock of the DefaultAccessorStrategy which will throw an UninitializedPropertyException.

Reply:
I would propose to move typed properties into separate fixtures classes - see: https://github.com/schmittjoh/serializer/pull/1330 for reference.

Reply:
OK thanks, I've implemented a new test case.